tests/installation/ipxe_install.pm

```diff
@@ -132,6 +132,42 @@ END_BOOTSCRIPT
     set_ipxe_bootscript($bootscript);
 }
 
+sub set_bootscript_agama {
+    my $host = get_required_var('SUT_IP');
+    my $arch = get_required_var('ARCH');
+    my $mirror_http = get_required_var('MIRROR_HTTP');
+    my $openqa_hostname = get_var('OPENQA_HOSTNAME', 'openqa.suse.de');
+    my $agama_iso = get_required_var('ISO');
+
+    # trim all strings from variables to get rid of bogus whitespaces
+    $host =~ s/^\s+|\s+$//g;
+    $arch =~ s/^\s+|\s+$//g;
+    $mirror_http =~ s/^\s+|\s+$//g;
+    $openqa_hostname =~ s/^\s+|\s+$//g;
+    $agama_iso =~ s/^\s+|\s+$//g;
```

Comment:
Why this part ```trim all strings from variables to get rid of bogus whitespaces``` is necessary ?

Reply:
Good question, they are not, only make sure removing some bogus white spaces. let me remove them and do VRs again.